app/views/staff/searches/_form.html.haml

```diff
@@ -0,0 +1,8 @@
+.govuk-grid-row
+  .govuk-grid-column-full
+    %h2.govuk-heading-m
+      = t("form.activity_search.heading")
+
+    = form_with model: ActivitySearch.new, url: search_path, method: "get" do |f|
+      = f.govuk_text_field :query, name: :query, width: 20
```

Comment:
Could we add a more useful label here? Something like `Enter your search query`? We could then have a hint - something like `You can search by RODA, Delivery Partner, or BEIS identifier, or by the activity's title`

Reply:
Thanks, I've added this copy and rebased my commits.

Reply:
Sweet 👍